src/modules/Tickets/TicketController.ts

```diff
@@ -7,46 +7,68 @@ import {
   Post,
   Put,
   Delete,
+  JsonController,
 } from "routing-controllers";
 
 import { TicketServices } from "./TicketServices";
+import { Container } from "typedi";
+import { ITicket } from "@app/data/abstraction/entities/ITicket";
+import _ from "lodash";
+import { NotAllow } from "@app/decorator/notAllow";
 
 @Controller()
 export class TicketController {
+  serviceInstance = Container.get(TicketServices);
+
   @Get("/")
-  response() {
-    return { msg: "Welcome" };
+  async welcomeMessage() {
+    try {
+      return "Welcome";
+    } catch (error) {
+      return;
+    }
```

Comment:
is this support to error???
```suggestion
    return {message: "Welcome"}
```